gumby/experiment.py

```diff
@@ -55,6 +55,7 @@ def __init__(self, vars):
         self.my_id = None
         self.vars = vars
         self.all_vars = {}
+        self.server_vars = {}
```

Comment:
I needed a global random variable to add to the filename of a download in a scenario, mostly to force the creation of new infohashes on each run of the experiment. Without this using a non-isolated DHT experiment can get a DHT response based on a previous run. Since this global is not decided upon by any of the clients, and has to change with each run, I came to the conclusion that the server gets to push variables too. Hence the server vars.